test/tests/serial/setup.sh.bats

```diff
@@ -0,0 +1,150 @@
+load "${REPOSITORY_ROOT}/test/test_helper/common"
+
+# ! NEEDS SERIAL EXECTION - DO NOT RUN IN PARALLEL
+
+IMAGE_NAME=${NAME:-mailserver-testing:ci}
+TEST_NAME_PREFIX='checking setup.sh: '
```

Comment:
I have just realized this was not a new test, but a changed version of `serial/no_container.bats`?

Perhaps this was an error on my part, and I missed that it was renamed :grimacing:

Reply:
> I have just realized this was not a new test, but a changed version of `serial/no_container.bats`?

Indeed

> Perhaps this was an error on my part, and I missed that it was renamed :grimacing: 

No problem I guess :)

Reply:
I will remove this file with one of the next (and hopefully final) commits.